### ISSUE TITLE
Fails with `uninitialized constant AocRb::Cli::FileUtils (NameError)`

### ISSUE THREAD

**Issue by andyw8** [pacso/aoc_rb#9]
I think this is due to a missing `require "fileutils"`?

```
andyw8 % chruby 3.1.1
andyw8 % ruby -v
ruby 3.1.1p18 (2022-02-18 revision 53f5fc4236) [arm64-darwin21]
andyw8 % gem install aoc_rb

Successfully installed aoc_rb-0.2.1
1 gem installed
andyw8 % aoc new advent-of-code-aoc
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
/Users/andyw8/.gem/ruby/3.1.1/gems/aoc_rb-0.2.1/lib/aoc_rb/cli.rb:22:in `new': uninitialized constant AocRb::Cli::FileUtils (NameError)

      FileUtils.mkdir_p bin_dir
      ^^^^^^^^^
Did you mean?  FileTest
	from /Users/andyw8/.gem/ruby/3.1.1/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
	from /Users/andyw8/.gem/ruby/3.1.1/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
	from /Users/andyw8/.gem/ruby/3.1.1/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
	from /Users/andyw8/.gem/ruby/3.1.1/gems/thor-1.1.0/lib/thor/base.rb:485:in `start'
	from /Users/andyw8/.gem/ruby/3.1.1/gems/aoc_rb-0.2.1/lib/aoc_rb/cli.rb:48:in `<top (required)>'
	from <internal:/opt/rubies/3.1.1/lib/ruby/3.1.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from <internal:/opt/rubies/3.1.1/lib/ruby/3.1.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
	from /Users/andyw8/.gem/ruby/3.1.1/gems/aoc_rb-0.2.1/exe/aoc:4:in `<top (required)>'
	from /Users/andyw8/.gem/ruby/3.1.1/bin/aoc:25:in `load'
	from /Users/andyw8/.gem/ruby/3.1.1/bin/aoc:25:in `<main>'
```

**Comment by pacso** [comment]
I wonder if this is a new issue in 3.1? 🤔 

I'll have a play and see if I can recreate it 👍🏻

**Comment by pacso** [comment]
This should be fixed by #10 and available in 0.2.2 now.

**Comment by andyw8** [comment]
Great, thanks!
